app/test/admin/moderation_case_resolve_test.dart

```diff
@@ -81,14 +91,27 @@ void main() {
       'SHOUT Deleting object from public bucket: "packages/oxygen-1.2.0.tar.gz".',
       'SHOUT Deleting object from public bucket: "packages/oxygen-2.0.0-dev.tar.gz".',
     ], fn: () async {
+      final starTime = clock.now();
       final mc = await _prepare(apply: true);
+
+      // cleanup doesn't remove case prematurely
+      await _verifyCaseExistence(mc.caseId, true);
+      await adminBackend.deleteModerationCases(before: starTime);
+      await _verifyCaseExistence(mc.caseId, true);
+
+      // close case
       expect(
         await _close(
           mc.caseId,
           reason: 'The package violated our policy.',
         ),
         'moderation-applied',
       );
+
```

Comment:
An ideal test would run the actual cleanup task with a clock set 3 years into the future.

Not a blocker